torch/csrc/cuda/Module.cpp

```diff
@@ -208,7 +208,7 @@ PyObject * THCPModule_cudaCachingAllocator_raw_alloc(PyObject *_unused, PyObject
         "(ssize_t size, intptr_t stream);");
     return nullptr;
   }
-  ssize_t size = PyLong_AsSsize_t(size_o);
+  size_t size = PyLong_AsSsize_t(size_o);
```

Comment:
Let's use auto here as type needs to be signed
```suggestion
  auto size = PyLong_AsSsize_t(size_o);
```